ibm/resource_ibm_is_vpc.go

```diff
@@ -117,6 +120,30 @@ func resourceIBMISVPC() *schema.Resource {
 				Description:  "VPC name",
 			},
 
+			isVPCDefaultNetworkACLName: {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: InvokeValidator("ibm_is_vpc", isVPCDefaultNetworkACLName),
```

Comment:
since we have not added this validator 
https://github.com/IBM-Cloud/terraform-provider-ibm/blob/025d52ae4a89807cb6b082e4eeaa1ab159201026/ibm/resource_ibm_is_vpc.go#L357

Does the validation working

Reply:
Yes its working